internal/gui/gui.go

```diff
@@ -192,27 +190,6 @@ func handleVPNLinkButton(conf *visorconfig.V1) {
 	}
 }
 
-// GetAvailPublicVPNServers gets all available public VPN server from service discovery URL
-func GetAvailPublicVPNServers(conf *visorconfig.V1) []string {
```

Comment:
Was supposed to be used for a quick connect functionality in the Systray application. So you could use the Systray directly without having to open the browser. Can likely be kept and used when we improve the Systray application.

Reply:
Oh, I didn't know about it. So I'll added it back.